bookkeeper-server/src/test/java/org/apache/bookkeeper/proto/TestPerChannelBookieClient.java

```diff
@@ -295,4 +302,42 @@ public void safeRun() {
         eventLoopGroup.shutdownGracefully();
         executor.shutdown();
     }
+
+    /**
+     * Test that TCP user timeout is correctly set in EpollEventLoopGroup.
+     */
+    @Test
+    public void testEpollChannelTcpUserTimeout() throws Exception {
```

Comment:
I validated that it doesn't fail on MacOS because of the absence of epoll:

```
[INFO] Running org.apache.bookkeeper.proto.TestPerChannelBookieClient
[WARNING] Tests run: 1, Failures: 0, Errors: 0, Skipped: 1, Time elapsed: 2.619 s - in org.apache.bookkeeper.proto.TestPerChannelBookieClient
[INFO] 
[INFO] Results:
[INFO] 
[WARNING] Tests run: 1, Failures: 0, Errors: 0, Skipped: 1
[INFO] 
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  30.897 s
[INFO] Finished at: 2021-08-24T23:03:34+02:00
[INFO] ------------------------------------------------------------------------
```